andela/Ah-backend-guardians
# #162685672 Authors should be able to tag articles

**What does this PR do?**
This PR enables users to add to tags to their articles when they author them.

**Description of Task to be completed?**
The application should enable a user to be able to add a tag to an article.

**Link to pivotal tracker story**
[#162685672](https://www.pivotaltracker.com/story/show/162685672)

**How should this be manually tested?**
Before testing run these commands in cmd

`git clone https://github.com/andela/Ah-backend-guardians.git`

`cd Ah-backend-guardians`

`git checkout ft-tag-articles-162685672`

`virtualenv env`

`. env/bin/activate`

`pip install -r requirements.txt`

Create a postgres database and a `.env` file in your root directory and add the following configurations.
```
SECRET_KEY=andela_sjdndkjghdf78872

DB_NAME=your_database_name

DB_USER=database_user

DB_PASSWORD=database_password

DB_HOST=127.0.0.1

ALLOWED_HOSTS=.localhost, .herokuapp.com

EMAIL_PORT=587

EMAIL_HOST_USER = gauthorshaven@gmail.com

EMAIL_HOST_PASSWORD = guardiansauthors
```
Open Postman and type `http://localhost:8000/api/articles/` in URL text box and enter the data according to the body of this screenshot, running a **POST** request and the output you get back should be as in the response part of the screenshot.

![image](https://user-images.githubusercontent.com/38134382/51557786-ef455b00-1e8e-11e9-8c94-90428a9b0165.png)
